CPackLists.txt

```diff
@@ -30,6 +30,8 @@
             DESTINATION ${CMAKE_INSTALL_HEADERDIR}/g3log
             COMPONENT headers)
 
+   INSTALL( FILES ${PROJECT_SOURCE_DIR}/cmake/G3LOGConfig.cmake
+            DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/G3LOG)
```

Comment:
Is this the default install path for cmake on OSX?  It does not seem to be the default install location under Linux: https://cmake.org/Wiki/CMake:How_To_Find_Libraries